tests/dir.props

```diff
@@ -106,9 +118,10 @@
     <DnuRestoreSource>@(DnuSourceList -> '--source %(Identity)', ' ')</DnuRestoreSource>
     <DnuRestoreDirs>@(DnuRestoreDir -> '%(Identity)', ' ')</DnuRestoreDirs>
 
-    <DnuRestoreCommand>$(DnuRestoreCommand) "$(DotnetToolCommand)"</DnuRestoreCommand>
+    <DnuRestoreCommand>$(DnuRestoreCommand) $(DotnetToolCommand)</DnuRestoreCommand>
```

Comment:
Probably at some point.  Not with this change unless it is trivial, and I don't think it is.